distributed/control-plane/confs/instance-2/deployment.toml

```diff
@@ -310,3 +317,16 @@ task_cleanup_interval_minutes = "{{ .Values.wso2.apim.configurations.governance.
 {{- else }}
 # At least one scheduler value is not empty
 {{- end }}
+
+{{- if .Values.wso2.apim.configurations.sdk.supported_languages }}
+[apim.sdk]
+supported_languages = {{ toJson .Values.wso2.apim.configurations.sdk.supported_languages }}
```

Comment:
This means apim.sdk configuration block is only applied when apim.sdk.supported_languages is defined. However apim.sdk supports other configurations as well. Shall we fix this?